hw/bitstream/BUILD

```diff
@@ -9,7 +9,7 @@ package(default_visibility = ["//visibility:public"])
 # variable.  See //rules/bitstreams.bzl for more information.
 #
 # Example:
-#   bazel test //sw/device/silicon_creator/lib/drivers:fpga_cw310_hmac_functest --define bitstream=gcp
+#   bazel test //sw/device/silicon_creator/lib/drivers:hmac_functest --define bitstream=gcp
```

Comment:
see suggestion on ci/BUILD: hmac_functest is a target but it's a test_suite that includes the binary as tested on multiple platforms.

Reply:
Gotcha. Updated in latest push.